.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: dart
+sudo: false
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  - test: --platform vm
+  - test: --platform firefox
```

Comment:
There aren't very many tests in this package, so it'll probably be faster if we run the VM and Firefox in the same task.